src/test/groovy/graphql/Issue296.groovy

```diff
@@ -1,84 +0,0 @@
-package graphql
```

Comment:
It's hard to find tests with file names like "Issue 296". Moving tests in generic file names to a better home. 

In this case, the test file for Introspection.